repos/fdbt-site/cypress_tests/cypress/support/steps.ts

```diff
@@ -431,7 +441,7 @@ export const editPassengerTypePointToPointPage = () => {
     getServiceLinkToClick();
     cy.get('@serviceToClick').then((serviceToClick) => {
         clickElementById(serviceToClick.toString());
-            cy.get('[class=govuk-table__body]').find('a').click();
+            cy.get('[class=govuk-table__body]').find('a').eq(0).click();
```

Comment:
can we use getElementByClass here?

Reply:
[20c99e8](https://github.com/fares-data-build-tool/create-data/pull/649/commits/20c99e8f614873969530b03182eda7acb2108840)